pkg/datapath/loader/base.go

```diff
@@ -158,6 +160,19 @@ func addENIRules(sysSettings []sysctl.Setting) ([]sysctl.Setting, error) {
 		return nil, fmt.Errorf("unable to install ip rule for ENI multi-node NodePort: %w", err)
 	}
 
+	// Add rules for router (cilium_host).
+	info := node.GetRouterInfo()
+	cidrs := info.GetIPv4CIDRs()
+	routerIP := net.IPNet{
+		IP:   nodeAddressing.IPv4().Router(),
```

Comment:
Were you able to call `node.GetInternalIP()` to get rid of the additional param to this function?